cmake/developer_package/compile_flags/os_flags.cmake

```diff
@@ -449,6 +449,10 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     # Build with multiple processes
     ov_add_compiler_flags(/MP)
 
+    # Specifies both the source character set and the execution character set as UTF-8.
+    # For details, refer to link: https://learn.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-and-executable-character-sets-to-utf-8?view=msvc-170
+    add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
```

Comment:
maybe you can use `ov_add_compiler_flags` instead ?
I suppose it will populate flags for C++ compiler, but not for CUDA's nvcc